src/bucket/Bucket.cpp

```diff
@@ -25,7 +25,8 @@
 #include "util/XDRStream.h"
 #include "xdrpp/message.h"
 #include <Tracy.hpp>
-#include <cassert>
+//#include <cassert>
```

Comment:
1. Do not leave commented out include (this is what git is for)
2. Make sure to run `clang-format` for each commit to avoid weird churn
3. As you're touching includes it's a good time to ensure that we have proper separation (empty line) between include groups: we normally have "main include(s)" followed by our includes followed by system includes. In this specific case, we'd want the last group to be
```

#include <Tracy.hpp>
#include <fmt/format.h>
#include <future>
```

Reply:
this applies to all files/commits

Reply:
@MonsieurNicolas Thanks for the Review.

Aah yes, i think I reverted the commented out code.

I have the right version of the clang-format -- > I think that is why the pull-request build was successful. Will remember that in the future. thanks


I think the clang-format rearrange the includes or I'm wrong?
I will have a look and if not will revert and rearrange them manually.